src/styles/antisnake.scss

```
@@ -0,0 +1,33 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+
+  @media (max-width: 899px) {
+    grid-template-columns: 1fr 1fr ;
+  }
+
+  @media (max-width: 599px) {
+    grid-template-columns: 1fr;
+  }
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-width: 300px;
+    height: 300px;
```

will be better if you create vars for this
```suggestion
    min-width: $blockSize;
    height: $blockSize;
```